scripts/check_urls.py

```diff
@@ -17,6 +18,36 @@
 FQDN_PATHS = ['/', '/docs/']  # list of paths on domain to get sitemap from
 SITEMAP = 'sitemap.xml'  # name of sitemap file
 EXTENSIONS_TO_SCAN = ['md', 'html', 'rst', 'ipynb']  # file extensions to scan for URLs
+ABS_URL_EXTENSIONS_TO_SCAN = ['md', 'tsx', 'js', 'html']  # file extensions to scan for non-absolute URLs
+
+
+def check_non_absolute_urls(path: str, extensions: List[str], urls: List[str]) -> List[List[str]]:
+    """Find any occurence of non-absolute URLs in the specified path and file types, e.g. in a repo
+
+    :param path: The path to scan
+    :param extensions: The file extensions to scan, e.g. ['md', 'js']
+    :param urls: The URL to scan for
+
+    :return: List of URLs found and in which file, e.g. [['/docs/hello', '/src/pages/index.tsx']]
+    """
+
+    non_abs_urls_found_in_files = []
+    for filename in glob.glob(path, recursive=True):
+        if has_extension(filename, extensions):
+            with open(filename) as f:
+                contents = f.read()
+                for url in urls:
+                    # find all occurences of the URL and determine if it's non-absolute
+                    matches = re.findall("(.*?)\/" + url, contents)

```

Comment:
```suggestion
                    matches = re.findall(f'(.*?)\/{url}', contents)

```

use f-string and single quotes instead of concatenation